Classes/Issues/Comments/Details/IssueCommentDetailCell.swift

```diff
@@ -83,6 +93,7 @@ final class IssueCommentDetailCell: IssueCommentBaseCell, ListBindable {
 
         editedLabel.font = Styles.Fonts.secondary
         editedLabel.textColor = Styles.Colors.Gray.light.color
+        editedLabel.isAccessibilityElement = false
```

Comment:
Do we need to set everything to false if the parent is set to true?

Reply:
Don't think so no, we can change this.